### PR TITLE
Apply black style to some test_SeqIO* files , version 19.10b0.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 ...

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to the below files; standard black style changes.
test_SeqIO_PdbIO.py
test_SeqIO_SeqXML.py
test_SeqIO_Xdna.py
test_SeqIO_write.py

### CODE REVIEW

File: Tests/test_SeqIO_PdbIO.py [hunk 1]
```diff
@@ -70,14 +74,17 @@ def test_seqres_read(self):
...
             self.assertEqual(chain.id, "1A8O:A")
             self.assertEqual(chain.annotations["chain"], "A")
-            self.assertEqual(str(chain.seq),
-                             "MDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETLLVQNANPD"
-                             "CKTILKALGPGATLEEMMTACQG")
+            self.assertEqual(
+                str(chain.seq),
+                "MDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETLLVQNANPD"
+                "CKTILKALGPGATLEEMMTACQG",
+            )
```
Comment (peterjc) [inline Tests/test_SeqIO_PdbIO.py]: Can we have the string on one line, or is it just over the 88 line limit?

File: Tests/test_SeqIO_write.py [hunk 2]
```diff
@@ -34,43 +34,121 @@
...
+                "Identifiers in each MultipleSeqAlignment must be unique",
+            ),
+            (
+                ["phylip", "phylip-relaxed", "phylip-sequential"],
+                ValueError,
+                "Repeated name 'Beta' (originally 'Beta'), possibly due to truncation",
+            ),
+        ],
+    ),
+]
```
Comment (peterjc) [inline Tests/test_SeqIO_write.py]: Well that block is rather a lot longer now... but so be it.

File: Tests/test_SeqIO_PdbIO.py (hunk 1 above)
Comment (svalqui) [reply Tests/test_SeqIO_PdbIO.py]: it misses by one :(

**Comment by peterjc** [comment]
Sorry, we should have merged this a while ago. The conflicts in ``Tests/test_SeqIO_write.py`` are non-trivial. It would probably be simplest to recreate the branch? Or revert your changes to that file and leave just the three other files only?

**Comment by svalqui** [comment]
I'll redo these files,

**Pull request closed without merging**